### PR TITLE
feat(DI): Support for registering a single service

### PR DESCRIPTION
**Pull request by zhenlei520**

### CODE REVIEW

File: src/Extensions/Masa.Utils.Extensions.DependencyInjection/IServiceRegistrar.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Microsoft.Extensions.DependencyInjection;
+
+public interface IServiceRegistrar
```
Comment (doddgu) [inline src/Extensions/Masa.Utils.Extensions.DependencyInjection/IServiceRegistrar.cs:6]: Registrar -> Register

File: src/Extensions/Masa.Utils.Extensions.DependencyInjection/README.zh-CN.md [hunk 2]
```diff
@@ -52,6 +52,27 @@ public interface IRepository<TEntity> : IScopedDependency
     ```
     > 等价于 service.AddScoped<IUserService, UserService>();
 
+  * 如果不希望接口被多次添加，只希望接口只有一个实现类，则在希望添加的实现类上方增加[Dependency(ReplaceServices = true)]的特性
```
Comment (doddgu) [inline src/Extensions/Masa.Utils.Extensions.DependencyInjection/README.zh-CN.md:55]: 如果希望接口只有一个实现类，则在实现类上方增加`[Dependency(ReplaceServices = true)]`即可

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Extensions/Masa.Utils.Extensions.DependencyInjection/IServiceRegistrar.cs (hunk 1 above)
Comment (zhenlei520) [reply src/Extensions/Masa.Utils.Extensions.DependencyInjection/IServiceRegistrar.cs]: Processed

File: src/Extensions/Masa.Utils.Extensions.DependencyInjection/README.zh-CN.md (hunk 2 above)
Comment (zhenlei520) [reply src/Extensions/Masa.Utils.Extensions.DependencyInjection/README.zh-CN.md]: Processed

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**